src/utils/constants/apiEndPoints.jsx

```diff
@@ -1,6 +1,17 @@
-const SIGNUP_API = '/api/auth/signup'
-const LOGIN_API = '/api/auth/login'
-const GET_VIDEOS = '/api/videos'
-const GET_CATEGORIES = '/api/categories'
+const SIGNUP_API = "/api/auth/signup";
+const LOGIN_API = "/api/auth/login";
+const GET_VIDEOS = "/api/videos";
+const GET_CATEGORIES = "/api/categories";
+const POST_HISTORY = "/api/user/history";
+const POST_WATCH_LATER = "/api/user/watchlater";
+const DELETE_WATCH_LATER = "/api/user/watchlater/:videoId";

```

Comment:
appreciation - liked that you're maintaining a file for APIs

Reply:
Thank you :D